### PR TITLE
Expand timeouts on JDBCExtractionNamespaceTest

### PR DESCRIPTION
**Pull request by drcrallen**

### CODE REVIEW

File: extensions/namespace-lookup/src/test/java/io/druid/server/namespace/cache/JDBCExtractionNamespaceTest.java [hunk 1]
```diff
@@ -196,7 +196,7 @@ public Void withHandle(Handle handle) throws Exception
     );
   }
 
-  @Test(timeout = 500)
+  @Test(timeout = 50_000)
```
Comment (xvrl) [inline extensions/namespace-lookup/src/test/java/io/druid/server/namespace/cache/JDBCExtractionNamespaceTest.java]: For long-running test we typically default to 60s timeout (see ChainedExecutionQueryRunner for instance). I think it makes sense to use the same value everywhere. We should probably make that a constant at some point.

Comment (drcrallen) [inline extensions/namespace-lookup/src/test/java/io/druid/server/namespace/cache/JDBCExtractionNamespaceTest.java]: Fixed

**Comment by himanshug** [comment]
LGTM

**Comment by guobingkun** [comment]
The transient failure still exists.
Build URL: https://travis-ci.org/druid-io/druid/jobs/76322236
```
Running io.druid.server.namespace.cache.JDBCExtractionNamespaceTest
Tests run: 6, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 62.587 sec <<< FAILURE! - in io.druid.server.namespace.cache.JDBCExtractionNamespaceTest
testSkipOld[null](io.druid.server.namespace.cache.JDBCExtractionNamespaceTest)  Time elapsed: 60.05 sec  <<< ERROR!
java.lang.Exception: test timed out after 60000 milliseconds
	at io.druid.server.namespace.cache.NamespaceExtractionCacheManager.removeNamespaceLocalMetadata(NamespaceExtractionCacheManager.java:296)
	at io.druid.server.namespace.cache.NamespaceExtractionCacheManager.delete(NamespaceExtractionCacheManager.java:514)
	at io.druid.server.namespace.cache.OnHeapNamespaceExtractionCacheManager.delete(OnHeapNamespaceExtractionCacheManager.java:99)
	at io.druid.server.namespace.cache.JDBCExtractionNamespaceTest.testSkipOld(JDBCExtractionNamespaceTest.java:270)
```

**Comment by himanshug** [comment]
still fails.... https://travis-ci.org/druid-io/druid/jobs/76409311

```
io.druid.server.namespace.cache.JDBCExtractionNamespaceTest

testSkipOld[null](io.druid.server.namespace.cache.JDBCExtractionNamespaceTest)  Time elapsed: 60.064 sec  <<< ERROR!

java.lang.Exception: test timed out after 60000 milliseconds

	at io.druid.server.namespace.cache.NamespaceExtractionCacheManager.removeNamespaceLocalMetadata(NamespaceExtractionCacheManager.java:296)

	at io.druid.server.namespace.cache.NamespaceExtractionCacheManager.delete(NamespaceExtractionCacheManager.java:514)

	at io.druid.server.namespace.cache.OnHeapNamespaceExtractionCacheManager.delete(OnHeapNamespaceExtractionCacheManager.java:99)

	at io.druid.server.namespace.cache.JDBCExtractionNamespaceTest.testSkipOld(JDBCExtractionNamespaceTest.java:270)

Running io.druid.server.namespace.cache.NamespaceExtractionCacheManagerExecutorsTest

Tests run: 6, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 50.414 sec <<< FAILURE! - in io.druid.server.namespace.cache.NamespaceExtractionCacheManagerExecutorsTest

testDelete(io.druid.server.namespace.cache.NamespaceExtractionCacheManagerExecutorsTest)  Time elapsed: 50.027 sec  <<< ERROR!

java.lang.Exception: test timed out after 50000 milliseconds

	at io.druid.server.namespace.cache.NamespaceExtractionCacheManager.removeNamespaceLocalMetadata(NamespaceExtractionCacheManager.java:296)

	at io.druid.server.namespace.cache.NamespaceExtractionCacheManager.delete(NamespaceExtractionCacheManager.java:514)

	at io.druid.server.namespace.cache.OnHeapNamespaceExtractionCacheManager.delete(OnHeapNamespaceExtractionCacheManager.java:99)

	at io.druid.server.namespace.cache.NamespaceExtractionCacheManagerExecutorsTest.testDelete(NamespaceExtractionCacheManagerExecutorsTest.java:333)

	at 
```

**Pull request merged**